compiler/include/AstToText.h

```diff
@@ -125,6 +125,11 @@ class AstToText
   void                   appendFormalDefault(ArgSymbol* arg);
   bool                   isTypeDefault(Expr* expr)                     const;
 
+  //
+  // Helper function for appending domain calls with multiple arguments
+  //
+  void                   appendDomain(CallExpr* expr, bool printingType);
```

Comment:
It's declared before it is used in the file and is unlikely to be used outside of it, so the function could be static and not put into the .h file.  But that's a personal preference

Reply:
I guess making it static would be good as we are not checking for domain and directly appending. I'll make the changes and send it in some time.

Reply:
I guess if we make it static, then we might have a problem in appending `mText` as it seems to be non-static.

Reply:
Static just means the compiler can see all calls to it and so can make optimizations based on that knowledge, so far as I know

Reply:
Sorry, what I meant was the `mText` and other `appendExpr` does not come into this scope as the function. When I try this, then I get `error: ‘mText’ was not declared in this scope`.

Reply:
I'm not sure I'm following - my suggestion was to remove the update to AstToText.h, and add `static` to the definition of appendDomain where it is right now.  Why would that cause `mText` to stop being found?

Reply:
The first three method in AstToText.cpp will be
```cpp
static void appendDomain(CallExpr* expr, bool printingType)
{
  mText += "domain(";
```
So, it's telling me that `mText` is not in the scope of this function.

Reply:
Ohhhhhh, I get it.  This is a method.  Never mind then, I think the only thing missing is the update to the other future I pointed out